src/app/account/repo/checks/[id]/form.js

```diff
@@ -27,12 +35,13 @@ export function FormBadge({ src }) {
       >
         Add badge to your Repo&lsquo;s README to show the latest check status
       </label>
-      <div className="mt-2 flex rounded-md shadow-sm">
+      <div className="mt-2  rounded-md shadow-sm">
         <div className="relative flex flex-grow items-stretch focus-within:z-10">
-          <div className="pointer-events-none absolute inset-y-0 left-0 flex items-center pl-3">
-            <CheckBadgeIcon
+          <div className=" absolute inset-y-0 left-0 flex items-center pl-3 cursor-pointer">
```

Comment:
We have an `Input` component that has a `prepend` I was thinking that could be used? With some small tweaks for the onclick.

https://github.com/EddieHubCommunity/HealthCheck/blob/main/src/components/forms/Input.js#L37

Reply:
Extra space not required 
```suggestion
          <div className="absolute inset-y-0 left-0 flex items-center pl-3 cursor-pointer">
```